### PR TITLE
Georges/paper

### PR DESCRIPTION
**Pull request by gebelangsn**
add intro and background sections

### CODE REVIEW

File: report/main.tex [hunk 1]
```diff
@@ -47,23 +58,187 @@
 
 
 \section{Introduction}
+Recent developments in natural language processing have shown that in-context learning allows large language models (LLMs) to quickly adapt to new tasks by using natural language instructions and a few demostrations as a prompt.
```
Comment (kylrth) [inline report/main.tex]:
```suggestion
Recent developments in natural language processing have shown that in-context learning allows large language models (LLMs) to quickly adapt to new tasks by using natural language instructions and a few demonstrations as a prompt.
```

File: report/main.tex [hunk 2]
```diff
@@ -47,23 +58,187 @@
...
+Specifically, this investigations examines how using the ART framework~\cite{paranjape2023art}, we can create and teach the LLM to use tools for Long Document Summarization.
+
+In the ART paper, they generate automatic multi-step decompositions for new tasks by selecting decompositions of related tasks in the task library, and then select and use tools in the tool library alongside LLM generation.
+We simplify the framework and only focus on constructing few-shot prompts for long document summarization that learn information onto external tools. We leverage ART's program grammar, but create new tools specically designed for summarization of large documents splitted into chunks of text.
```
Comment (kylrth) [inline report/main.tex]:
```suggestion
We simplify the framework and only focus on constructing few-shot prompts for long document summarization that learn information onto external tools. We leverage ART's program grammar, but create new tools specially designed for summarization of large documents split into chunks of text.
```

Comment (kylrth) [reply report/main.tex]: or did you mean `specifically`?

File: report/main.tex [hunk 3]
```diff
@@ -47,23 +58,187 @@
...
+
+
+\section{Background}
+Apart from the ART paper, another work that uses tools with an LLM is Toolformer \add{cite}.
+However, different from ART, Toolformer was finetuned to learn how to use tools, as opposed to in-context learning. The advantage of our approach is that you can switch LLM without the need of finetuning.
```
Comment (kylrth) [inline report/main.tex]:
```suggestion
However, different from ART, Toolformer was finetuned to learn how to use tools, as opposed to relying on in-context learning. The advantage of our approach (and ART in general) is that you can switch LLMs without need of finetuning.
```

File: report/main.tex [hunk 4]
```diff
@@ -47,23 +58,187 @@
...
+
+Other relevant recent work proposes prompting LLMs to mimic a chain of thought (CoT) for multi-step reasoning by stating in the prompt to think Step-by-step \add{cite}.
+
+Finally, state-of-the-art approaches for long document summarization include the following:
+Top Down Transformer (Long Document Summarization with Top-down and Bottom-up Inference)
+LongT5: Efficient Text-to-Text Transformer for Long Sequences
+Pegasus X: Investigating Efficiently Extending Transformers for Long Document Summarization and Document Summarization with Text Segmentation \add{\cite}.
```
Comment (kylrth) [inline report/main.tex]: This sentence doesn't feel like a sentence, is it supposed to be a bulleted list?

**Review by kylrth — CHANGES_REQUESTED** [review]

File: report/main.tex (hunk 4 above)
Comment (gebelangsn) [reply report/main.tex]: I converted it into a sentence.. let me know if it looks good

**Pull request closed without merging**